custom/icds_reports/migrations/0193_add_fields_aww_inactivity_report.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.28 on 2020-05-26 20:51
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0192_add_child_beneficiary_status_fields_to_chm'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='aggregateinactiveaww',
+            name='no_of_days_inactive',
+            field=models.PositiveIntegerField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name='aggregateinactiveaww',
+            name='no_of_days_since_start',
+            field=models.PositiveIntegerField(blank=True, null=True),
+        ),
+    ]
```

Comment:
it would also be helpful to check if this table have the required indexes present. If not, we should create them with this migration. It will make the query faster.